codesystems/CodeSystem-England-MedicationRequestControlledDrug.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CodeSystem xmlns="http://hl7.org/fhir">
+    <id value="England-MedicationRequestControlledDrug"/>
+    <url value="https://fhir.nhs.uk/CodeSystem/England-MedicationRequestControlledDrug"/>
+    <identifier>
+        <system value="https://fhir.nhs.uk/identifier/CodeSystem"/>
+        <value value="medicationrequest-controlled-drug"/>
+    </identifier>
+    <version value="1.0.0"/>
+    <name value="EnglandMedicationRequestControlledDrug"/>
+    <title value="England MedicationRequestControlledDrug"/>
```

Comment:
MedicationRequestControlledDrug -> MedicationRequest Controlled Drug

Reply:
Done with the required changes.